packages/cra-template/README.md

```diff
@@ -0,0 +1,7 @@
+# @fluentui/cra-template
+
+**Cra for [Fluent UI React](https://developer.microsoft.com/en-us/fluentui)**
+
+CRA template for FluentUI Web
+
+TODO: Add more details!
```

Comment:
uuuuh nit: add more details ? :-)

Reply:
yeah lets copy this https://github.com/facebook/create-react-app/blob/master/packages/cra-template-typescript/README.md